src/rust/lib/data/src/text.rs

```diff
@@ -152,6 +167,17 @@ impl Span {
         Span {index:Index::new(0), size}
     }
 
+    /// Get the index of the last character in the span.
+    ///
+    /// If the span is empty or starts at 0, returns `None`.
```

Comment:
```suggestion
    /// If the span is empty or ends at 0, returns `None`.
```

Reply:
I've removed that part, as you mentioned that this situation is impossible.